tests/testthat/test-call_derivation.R

```diff
@@ -2,7 +2,9 @@ library(admiral.test)
 data(admiral_ae)
 data(admiral_vs)
 
-test_that("call_derivation works", {
+## Test 1: Test that call_derivation generates expected summary output  ----
+# ---- call_derivation, test 1: call_derivation Test 1:  Test that call_derivation generates expected summary output ----
+test_that("call_derivation, test 1: call_derivation Test 1:  Test that call_derivation generates expected summary output", {
   input <- admiral_vs[sample(seq_len(nrow(admiral_vs)), 1000), ]
```

Comment:
```suggestion
  input <- admiral.test::admiral_vs[sample(seq_len(nrow(admiral.test::admiral_vs)), 1000), ]
```

Reply:
Hey @jeffreyad.  Sorry this one is taking a while.  Can you remove the `library(admiral.test)` and `data(...)` at the top of the test file and replace all the calls for` admiral_vs`, etc with `admiral.test::admiral_vs` and the similar calls